### ISSUE TITLE
[BUG] Can't prove queue/create.c with VeriFast

### ISSUE THREAD

**Issue by master-q** [FreeRTOS/FreeRTOS#865]
**Describe the bug**
Can't prove queue/create.c with VeriFast.

**Target**
- Development board: Any
- Instruction Set Architecture: Any
- IDE and version: VeriFast 19.12 for C and Java (released 2019-12-02)
- Toolchain and version: VeriFast 19.12 for C and Java (released 2019-12-02)

**Host**
- Host OS: Debian GNU/Linux amd64
- Version: unstable

**To Reproduce**
- git clone git@github.com:FreeRTOS/FreeRTOS.git
- cd FreeRTOS/FreeRTOS/Test/VeriFast
- make

**Expected behavior**
We should see following message:

```
0 errors found (??? statements verified)
```

**Screenshots**

```
$ verifast -version|&grep released
VeriFast 19.12 for C and Java (released 2019-12-02) <https://github.com/verifast/verifast/>
$ make
queue/create.c
include/proof/queue.h(187,2-3): Parse error.
Coverage regression failed: Expected 325 statements verified.
make: *** [Makefile:21: queue] Error 1
```

```
$ vfide -I include queue/create.c
```
![verifast_error](https://user-images.githubusercontent.com/277488/197916059-85cec03b-6b0d-47df-a394-7045c367035a.png)

**Additional context**
None.

**Comment by master-q** [comment]
I believe this is caused by the miss style comments.
Attached patch works for queue/create.c.

[fix_miss_style_comments.patch.txt](https://github.com/FreeRTOS/FreeRTOS/files/9866407/fix_miss_style_comments.patch.txt)

May I create PR to fix this issue?

**Comment by n9wxu** [comment]
We welcome any PR's. Please let me know when it is ready for review.

**Comment by n9wxu** [comment]
This has been resolved with PR #836
